states/networks.sls

```diff
@@ -54,9 +54,9 @@ libvirt_virsh_net_startstop_{{ name }}:
     - run
   {% if n.ensure|default('running') == 'running' %}
     - name: virsh net-start {{ name }}
-    - unless: virsh -q net-list --all | grep -Eq '^{{ name }}\s+active'
+    - unless: virsh -q net-list --all | grep -Eq '^\s?{{ name }}\s+active'
```

Comment:
please replace ``\s+`` by ``\s*``, to make backwards-compatiblity working (in case there's no space between ^ and ``{{ name }}``)